packages/components/organisms/f-header/src/tenants/nb-NO.js

```diff
@@ -56,8 +56,8 @@ export default {
         url: '/account/logout',
         gtm: 'click_logout'
     },
-    offers: {
-        text: '%OFFERS%',
-        url: '/offers'
-    }
+    selectYourCountryText: 'Velg land',
```

Comment:
Should these be inside their own object, i.e., `countrySelector.selectYourCountryText`?